includes/class-stats.php

```diff
@@ -28,7 +28,43 @@ private function __construct() {
 	 * Do initialization of all the things needed for stats.
 	 */
 	public function init() {
-		// Do init stuff.
+		$this->initialize_wpdb();
+	}
+
+	/**
+	 * Initialize the alias for the stats table on the wpdb object.
+	 *
+	 * @return void
+	 */
+	private function initialize_wpdb() {
+		global $wpdb;
+		$wpdb->job_manager_stats = $wpdb->prefix . 'job_manager_stats';
+		$wpdb->tables[]          = 'job_manager_stats';
+	}
+
+	/**
+	 * Migrate the stats table to the latest version.
+	 *
+	 * @return void
+	 */
+	public function migrate() {
+		global $wpdb;
+		$collate = $wpdb->get_charset_collate();
+		require_once ABSPATH . 'wp-admin/includes/upgrade.php';
+		\dbDelta(
+			[
+				"CREATE TABLE {$wpdb->job_manager_stats} (
+				`id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+				`date` DATE NOT NULL,
```

Comment:
I had to remove the `CURRENT_DATE` here because apparently it is not supported by MySQL older than the 8.0.20